deckhouse-controller/pkg/controller/deckhouse-release/updater/updater.go

```diff
@@ -160,3 +183,23 @@ func (api *KubeAPI) SaveReleaseData(ctx context.Context, release *v1alpha1.Deckh
 		NotifiedAnnotation:   strconv.FormatBool(data.Notified),
 	})
 }
+
+func (api *KubeAPI) IsKubernetesVersionAutomatic(ctx context.Context) (bool, error) {
```

Comment:
```suggestion
func (api *KubeAPI) IsKubernetesVersionAutomatic(ctx context.Context) error {
```
looks like it will be better without boolean return value. maybe return named error?